src/clincoded/static/components/variant_central/interpretation/basic_info.js

```diff
@@ -209,11 +209,23 @@ var CurationInterpretationBasicInfo = module.exports.CurationInterpretationBasic
     //Render RefSeq or Ensembl transcripts table rows
     renderRefSeqEnsemblTranscripts: function(item, key, source) {
         // Only if nucleotide transcripts exist
-        if (item.hgvsc && item.source === source) {
+        if (item.source === source) {
+            const isCanonicalTranscript = item.canonical && item.canonical === 1;
+            
+            // only enable MANE transcript label in RefSeq Transcripts section
+            const isMANETranscript = !!item.mane;
+
             return (
-                <tr key={key} className={(item.canonical && item.canonical === 1) ? 'primary-transcript' : null}>
-                    <td className="hgvs-term"><span className="title-ellipsis">{item.hgvsc}</span></td>
-                    <td>{(item.exon) ? item.exon : '--'}</td>
+                <tr key={key} className={isCanonicalTranscript || isMANETranscript ? "marked-transcript" : null}>
+                    <td className="hgvs-term">
+            <span className="title-ellipsis">{item.hgvsc || item.transcript_id} {item.gene_symbol && `(${item.gene_symbol})`}</span>
+                        {/* show label for canonical transcript */}
```

Comment:
Indent the <span> to line up other rows under <td> tag.

Reply:
Good catch, that line is too much in front..